static/js/components/UserInfoCard.js

```diff
@@ -26,8 +27,45 @@ export default class UserInfoCard extends React.Component {
     </div>
   );
 
+  renderAboutMeWithData = (profile: Profile): Array<React$Element<*>> => ([
+    <h3 className="heading" key="heading">About Me</h3>,
+    <div className="bio" key="bio">{profile.about_me}</div>
```

Comment:
when react elements are declared in an array they must have unique `key` props passed in.